mpd/mpd_read_write_test.go

```diff
@@ -489,3 +489,20 @@ func TestWriteToFileTruncate(t *testing.T) {
 	xmlStr = testfixtures.LoadFixture(out)
 	testfixtures.CompareFixture(t, "fixtures/truncate_short.mpd", xmlStr)
 }
+
+func TestReadComment(t *testing.T) {
```

Comment:
for both of these tests please add a full manifest to ensure the read / write of a full MPD manifest still works as is or try modifying one of the existing fixtures with that full manifest where other properties are being compared and validated as well.